src/mon/PGMap.cc

```diff
@@ -69,6 +70,7 @@ void PGMapDigest::encode(bufferlist& bl, uint64_t features) const
   encode(avail_space_by_rule, bl);
   encode(purged_snaps, bl);
   encode(osd_sum_by_class, bl, features);
+  encode(max_raw_used_by_rule, bl);
```

Comment:
OK, the schema change at the encoder side looks sane – it's the classical extension-at-the-end.